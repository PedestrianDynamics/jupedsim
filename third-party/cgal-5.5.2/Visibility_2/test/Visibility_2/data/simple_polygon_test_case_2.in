# Test case 2 for simple polygons. See https://cgal.geometryfactory.com/CGAL/Members/wiki/Visibility/TestCases
4
0 0
4 4
8 4
0 8
4
0 1
1 2
2 3
3 0
1 3