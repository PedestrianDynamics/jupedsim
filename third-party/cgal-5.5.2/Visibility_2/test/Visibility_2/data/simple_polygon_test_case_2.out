4
0 0
4 4
32/5 24/5
0 8
4
0 1
1 2
2 3
3 0